pkg/util/strings.go

```
@@ -33,6 +33,7 @@ func stringsFallback(vals ...string) string {
 
 // SplitString splits a string and returns a list of strings. It supports JSON list syntax and strings separated by commas or spaces.
 // It supports quoted strings with spaces, e.g. "foo bar", "baz".
+// It will return an empty list if it fails to parse the string.
 func SplitString(str string) []string {
```

I think this function can call the new one `SplitStringWithError()` and ignore the error, something like:
```
func SplitString(str string) []string {
  result, _ := SplitStringWithError(str)
  return result
}
```
Or remove it completely and use the new one in all places.